charts/brig/templates/configmap.yaml

```diff
@@ -150,6 +150,19 @@ data:
       configTTL: 3600 # 1 hour
       tokenTTL: 43200 # 12 hours
 
+    {{- if .sft }}
+    {{- with .sft }}
+    sft:
+      sftBaseDomain: {{ required "Missing value: .sft.sftBaseDomain" .sftBaseDomain }}
+      {{- if .sftSRVServiceName }}
+      sftSRVServiceName: {{ .sftSRVServiceName }}
+      {{- end }}
+      {{- if .sftDiscoveryIntervalSeconds }}
+      sftSRVServiceName: {{ .sftDiscoveryIntervalSeconds }}
```

Comment:
This is a duplicate key.

Also, I wonder why we assemble `sftBaseDomain` and `sftSRVServiceName` in the backend, and not just pass the fully qualified record name to the server.